lib/lookbook/entities/preview_example.rb

```diff
@@ -57,6 +57,10 @@ def url_path
       lookbook_inspect_path(path)
     end
 
+    def preview_path
```

Comment:
For consistency I think this should be added to the `PreviewExample` class as well as they are intended to be API-compatible where it makes sense to be.

I also think this method should probably accept a params hash that is passed to the path helper - obviously not useful for the JSON generation but then this method could be used in the views to replace calls like this: https://github.com/ViewComponent/lookbook/blob/f3cffecd9a317efba0e54bf5c10b27cba3fa7757/app/views/lookbook/inspector/show.html.erb#L54

Which again would make things more consistent.

Reply:
@allmarkedup did you mean a different class? This is `PreviewExample`.

Reply:
Urgh sorry @joelhawksley - yes I meant `PreviewGroup`!